stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -1008,24 +979,51 @@ public func _tffunc<In : TensorGroup, Out : TensorGroup>(
   return traceContext.specializeTFFunction(with: [])
 }
 
-/// Returns a valid TF device string such as
-/// "/job:localhost/replica:0/task:0/device:CPU:0", which corresponds to the
-/// closest enclosing withDevice() construct.
-/// A return value of nil indicates the absence of withDevice().
 internal extension _ExecutionContext {
-  @usableFromInline
+  /// Returns a valid TensorFlow device name such as, which corresponds to the
+  /// closest enclosing withDevice() call.
```

Comment:
Some minor nit on API doc comments. Swift functions are identified by their full name, which includes argument labels. Therefore, we often use the full name in doc comments.

```suggestion
  /// closest enclosing `withDevice(_:perform:)` call.
```